packages/components/src/ResourcePicker/Toolbar/SortOptions/SortOptions.component.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { translate } from 'react-i18next';
+import OrderChooser from './OrderChooser';
+import I18N_DOMAIN_COMPONENTS from '../../../constants';
+import getDefaultT from '../../../translate';
+
+import theme from './SortOptions.scss';
+
+export const TYPES = {
+	NAME: 'name',
+	DATE: 'date',
+};
+
+function SortOptions({ t, types, onChange, nameAsc, dateAsc }) {
+	return (
+		!!types.length && (
+			<div className={classNames('sort-options', theme['sort-options'])}>
+				<span className={classNames('option-label', theme['option-label'])}>
+					{t('SORT', { defaultValue: 'Sort:' })}
+				</span>
+				{types.includes(TYPES.NAME) && (
+					<OrderChooser
+						icon={'talend-sort-az'}
+						label={t('SORT_BY_NAME', { defaultValue: 'Sort by name' })}
+						asc={nameAsc}
+						onClick={() => onChange(TYPES.NAME, !nameAsc)}
+					/>
+				)}
+				{types.includes(TYPES.DATE) && (
+					<OrderChooser
+						icon={'talend-sort-desc'}
+						label={t('SORT_BY_DATE', { defaultValue: 'Sort by date' })}
+						asc={dateAsc}
+						onClick={() => onChange(TYPES.DATE, !dateAsc)}
+					/>
+				)}
+			</div>
+		)
+	);
+}
+
+SortOptions.propTypes = {
+	t: PropTypes.func,
+	onChange: PropTypes.func,
+	nameAsc: PropTypes.boolean,
+	dateAsc: PropTypes.boolean,
+	types: PropTypes.array,
```

Comment:
Proposal : what about flags to enable them ?
```javascript
<SortOptions {...props} useName useDate />
```
This makes props have simple names, and avoid exposing the array of 2 constants

If not, here you can use `PropTypes.oneOf`